spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/app/AppDeployer.java

```diff
@@ -66,6 +67,21 @@
 	static final String INSTANCE_INDEX_PROPERTY_KEY = "INSTANCE_INDEX";
 
 	/**
+	 * The deployment property for the memory setting for the container that will run the app.
+	 * The memory is specified in megabytes with optional trailing 'm' like '1024m'. How this property
```

Comment:
From this PR: https://github.com/spring-cloud/spring-cloud-deployer/pull/139, `memory` option correlates to JVM max heap size `-Xmx`. But,
>  The memory is specified in megabytes with optional trailing 'm' like '1024m'
this would conflict with the JVM heap size as the default is specified as `bytes` for the heap size.